src/controller/build_info.h.in

```diff
@@ -29,9 +29,7 @@ constexpr unsigned int NST_VERSION =
     @NST_V_MAJOR@ * 1000 + @NST_V_MINOR@ * 100 + @NST_V_PATCH@;
 
 constexpr char PROGRAM_BUILD_INFO[]=
-    "@NST_VERSION_FULL@\n"
-    "built on @CMAKE_SYSTEM@\n"
-    "by C++ compiler @CMAKE_CXX_COMPILER_ID@ @CMAKE_CXX_COMPILER_VERSION@";
```

Comment:
Hmm, you have reverted this bit in f517017226845c2d9ed111e3519791f4b9b9eced. Was that intentional or accidental?